tools/mozilla/buildxpi.py

```diff
@@ -66,21 +65,20 @@ def __init__(self, message=None, **kwargs):
 
     def __str__(self):
         """Format exception message string (avoiding TypeErrors)"""
-        output = ''
+        output = ""
```

Comment:
IMHO it would be easier to review if this specific change of quoting happens in a single commit, with no other changes mixed.

Reply:
Okay, I've split this into three commits - adding black, applying black without string normalization and string normalization.